test/brsTypes/components/LayoutGroup.test.js

```diff
@@ -0,0 +1,39 @@
+const brs = require("brs");
+const { LayoutGroup } = brs.types;
+
+describe("Rectangle", () => {
```

Comment:
don't forget to update this name 🌮

Reply:
Ugh, I need to write a script to do this for me or something so I don't miss silly copy changes 😅